### DISCUSSION TITLE
How to close figures of Decay chain diagrams?

### DISCUSSION THREAD

**Discussion by CLargeron** [radioactivedecay/radioactivedecay#90] [Q&A]
The Nuclide class includes a plot() method for drawing decay chain diagrams. However, I don't understand how i can close these plot
such as plt.close() or plt.clf

nuc = rd.Nuclide('Mo-99')
nuc.plot()

here, plot is a module part of nuclides but there is noway to close these diagrams. Dis I miss something ? 

Thank you very much,

**Comment by alexmalins** [comment]
Hi there!
Can you please share more about the operating system, python environment, and exact code you are running?
I'm guessing you are running the code in a Python terminal like:
```
>>> import matplotlib.pyplot as plt
>>> import radioactivedecay as rd
>>> nuc = rd.Nuclide('Mo-99')
>>> nuc.plot()
(<Figure size 450x750 with 1 Axes>, <AxesSubplot:>)
>>> plt.show()
```
Then a window pops up with the diagram, but you cannot enter further Python commands in the terminal unless you close the window?
![image](https://github.com/radioactivedecay/radioactivedecay/assets/22991362/15523369-29cf-4971-b0df-8f60e3c9d5d9)
Is that right?

If instead you write:
```
>>> plt.show(blocking=False)
```
You should be able to enter new Python commands without closing the figure. You can then run:
```
>>> plt.close()
```
to close the figure window from Python.

Does this help? You can read more about this topic in the matplotlib [show](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.show.html) and [close](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.close.html) documentation.
